blazeutils/spreadsheets.py

```diff
@@ -252,33 +279,70 @@ def nextrow(self):
         self.colnum = 0
 
 
+class WriterX:
+    @property
+    def ws(self):
+        return self._writer.ws
+
+    @property
+    def rownum(self):
+        return self._writer.rownum
+
+    @rownum.setter
+    def rownum(self, value):
+        self._writer.rownum = value
+
+    @property
+    def colnum(self):
+        return self._writer.colnum
+
+    @colnum.setter
+    def colnum(self, value):
+        self._writer.colnum = value
+
+    def __init__(self, ws):
+        self._writer = None
+        if openpyxl:
+            from openpyxl.worksheet.worksheet import Worksheet
+            if type(ws) == Worksheet:
```

Comment:
We should use `isinstance` here, so we can match on subclasses of these worksheets.